packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart

```diff
@@ -366,6 +366,14 @@ void runTests() {
     );
     final GoogleMapController controller = await controllerCompleter.future;
 
+    await tester.pumpAndSettle();
+
+    // TODO(mossmana): Adding this delay addresses
+    // https://github.com/flutter/flutter/issues/131783. It may be related
+    // to https://github.com/flutter/flutter/issues/54758 and should be
+    // re-evaluated when that issue is fixed.
+    await Future<void>.delayed(const Duration(seconds: 1));
```

Comment:
This change was added to the Android implementation in https://github.com/flutter/packages/pull/4768. I'm not sure why it only started failing now, but this should ideally fix this.